src/components/StatisticsCard/Battles.js

```diff
@@ -3,10 +3,11 @@ import { OverlayTrigger, Tooltip as Tips } from 'react-bootstrap';
 import BarChart from '../Charts/BarChart';
 
 const Battles = (props) => {
-  const lastSnapshot = props.tankStats.data.snapshots.at(-1);
+  const stats = props.tankStatsCard ? props.tankStatsCard : props.accountStats?.data;
```

Comment:
To simplify the code, you can utilize the logical OR (||) operator
```suggestion
  const stats = props.tankStatsCard || props.accountStats?.data;
```